packages/react/src/components/Footer/LegalNav.js

```diff
@@ -30,6 +44,7 @@ const LegalNav = ({ links }) => {
       <nav className={`${prefix}--legal-nav`}>
         <ul className={`${prefix}--legal-nav__list`}>
           {renderListItems(links)}
+          {renderTrusteItem()}
```

Comment:
I'm not crazy about having an empty `li` in the footer at all times. This also renders the `:last-child` styling here pointless unless the link is injected.